styles.css

```diff
@@ -0,0 +1,509 @@
+    background-size: 100% 100%;
+}
+
+.article-inner-container-text p {
+    font-size: 1.5rem;
+    padding: 45px 0 10px 0;
+}
+
+
+.article-inner-container-text p a {
+    text-decoration: none;
+    color: #5722CD;
+    position: relative;
+}
+
+
+.article-inner-container-text p a::after {
+    content: "";
+    width: 15%;
+    height: 2px;
+    background-color: #5722CD;
+    transition: width .35s ease-in-out;
+    position: absolute;
+    left: 165px;
+    top: 15px;
+}
+
+.article-inner-container-text p a:hover::after {
+    width: 60%;
+}
+
+.driehoek-drie{
+    width: 100vw;
+    height: 30px;
+    position: absolute;
+    bottom: 0;
+    border-right: 100vw solid transparent;
+    border-bottom: 30px solid white;
+}
+
+.section-two-section::before {
+    content: 'work';
+    position: absolute;
+    top: 0;
+    font-size: 25rem;
+    z-index: -1;
+    color: white;
+}
+
+/* --------------- 4.4 main articles ------------------- */
+.section-three-section {
+    display: flex;
+    justify-content: center;
+    padding: 325px 0 325px 0;
+    position: relative;
+    align-items:center;
+}
+
+.main-section-three-outer-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    flex-basis: 1650px;
+    flex-wrap: wrap;
+    gap: 50px;
+
+}
+
+.article-section-three {
+    border: 2px solid #5722CD;
+    flex-basis: 500px;
+    box-shadow: 0 15px 10px rgba(0, 0, 0, 0.1), 0 -5px 15px rgba(0, 0, 0, 0.06);
+    height: 500px;
+    z-index: 1;
+    background-color: white;
+}
+
+.article-section-three img {
+    width: 150px;
+    padding: 40px 0 0 40px;
+}
+
+.article-section-three h3 {
+    color: #5722CD;
+    padding: 10px 0 30px 40px;
+    font-size: 2.1rem;
+    font-weight: bolder;
+}
+
+.article-section-three p {
+    padding: 0 30px 0 40px;
+    color: #7A7A7A;
+    font-size: 1.4rem;
+    font-weight: 400;
+}
+
+.section-three-section::before {
+    content: 'services';
+    position: absolute;
+    top: 3%;
+    font-size: 20rem;
+    z-index: -1;
+    color: #F3F6F9;
+}
+
+
+/* --------------- 4.5 main formulier ------------------- */
+
+.section-formulier {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    background-color: #4845E4;
+    padding: 100px 0 150px 0;
+}
+
+.formulier-container {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 1300px;
+    align-items: center;
+}
+
+form {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%
+}
+
+.formulier-container h2 {
+    font-size: 4rem;
+    color: #F3F6F9;
+}
+
+.button-formulier {
+    text-align: center;
+    margin: 40px 0 0 0;
+    padding: 35px 100px 35px 100px;
+    border-radius: 50px;
+    font-size: 1.2rem;
+    color: white;
+    background-color: #ED4385;
+    box-shadow: 0 10px 10px rgba(0, 0, 0, 0.01), 0 -5px 15px rgba(0, 0, 0, 0.06);
+    text-transform: uppercase;
+}
+
+.form-invoerveld {
+    width: 70%;
```

Comment:
Dit kun je beter vermijden! Zorg dat de input-velden altijd 100% breedte hebben en dat ze beperkt worden door de plek waar ze in staan, dus het formulier of de inner-container van dit gedeelte bepaald hoeveel ruimte er beschikbaar is!